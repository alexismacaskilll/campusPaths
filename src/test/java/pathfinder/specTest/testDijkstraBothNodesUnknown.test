# Test the Find Path with two nodes that are not in the graph

# Create graph
CreateGraph simpleGraph

# Add nodes to the graph
AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

# Add edges to the graph
AddEdge simpleGraph A C 10.0
AddEdge simpleGraph A B 1.0
AddEdge simpleGraph B C 2.0

# Find shortest path from E to F
FindPath simpleGraph E F