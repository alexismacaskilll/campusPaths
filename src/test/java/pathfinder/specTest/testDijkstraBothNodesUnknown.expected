# Test the Find Path with two nodes that are not in the graph

# Create graph
created graph simpleGraph

# Add nodes to the graph
added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

# Add edges to the graph
added edge 10.000 from A to C in simpleGraph
added edge 1.000 from A to B in simpleGraph
added edge 2.000 from B to C in simpleGraph

# Find shortest path from E to F
unknown node E
unknown node F